.travis.yml

```diff
@@ -27,6 +32,34 @@ script:
 
 jobs:
   include:
+    - stage: Smoke Testing
+      php: 7.3
+      env: DB=sqlite
+
+    - stage: Smoke Testing
+      if: type = pull_request
+      env: DB=none PULL_REQUEST_CODING_STANDARDS
+      php: 7.1
```

Comment:
We could remove this line since Travis uses the lowest PHP version in the matrix by default. I know you just moved stuff but that would save us a bit of work in the future (when dropping PHP 7.1 for example :smile:)